src/views/List.jsx

```diff
@@ -12,6 +19,7 @@ export function List({ data }) {
 				 * using the `ListItem` component that's imported at the top
```

Comment:
certainly it doesn't affect performance, but for future you or another team member, it will be helpful to remove this TODO comment and surrounding curly braces. That way we will all know we don't need to come back to this TODO ✅